### ISSUE TITLE
FreeBSD - Task does not exist

### ISSUE THREAD

**Issue by jaredrsowers** [d0u9/youtube-dl-webui#17]
I am running this in a on FreeBSD and I keep getting "Task does not exist" errors, pretty much with each download. I have to Ctrl-C to exit and then relaunch it... Any ideas?

```
Task does not active, tid=3ad1a58c6edd310c590a989edac902188f0c0b7e
Task does not active, tid=3ad1a58c6edd310c590a989edac902188f0c0b7e
Traceback (most recent call last):
  File "/usr/local/bin/youtube-dl-webui", line 11, in <module>
    load_entry_point('youtube-dl-webui==0.2.1', 'console_scripts', 'youtube-dl-webui')()
  File "/usr/local/lib/python3.6/site-packages/youtube_dl_webui-0.2.1-py3.6.egg/youtube_dl_webui/__init__.py", line 27, in main
    core.start()
  File "/usr/local/lib/python3.6/site-packages/youtube_dl_webui-0.2.1-py3.6.egg/youtube_dl_webui/core.py", line 289, in start
    self.msg_mgr.run()
  File "/usr/local/lib/python3.6/site-packages/youtube_dl_webui-0.2.1-py3.6.egg/youtube_dl_webui/msg.py", line 78, in run
    cb(svr, evnt, data, arg)
  File "/usr/local/lib/python3.6/site-packages/youtube_dl_webui-0.2.1-py3.6.egg/youtube_dl_webui/core.py", line 215, in event_progress
    cls._task_mgr.progress_update(tid, data)
  File "/usr/local/lib/python3.6/site-packages/youtube_dl_webui-0.2.1-py3.6.egg/youtube_dl_webui/task.py", line 266, in progress_update
    raise TaskInexistenceError('task does not exist')
youtube_dl_webui.utils.TaskInexistenceError: 'Task does not exist'
```

**Comment by d0u9** [comment]
Well, Give me some time to install a freeBSD for test.

**Comment by carlosedp** [comment]
I'm running on a Raspberry Pi 3 with Linux Stretch (Raspbian) on a Docker container (replaced the ffmpeg binary with armel one) and see a similar problem, after I add a video to it, the download finishes correcly but the interface freezes until I restart and this is printed to the logs:
```
  INFO -  worker.py:122 - start downloading, url - https://www.youtube.com/watch?v=8w2lNpixqOc
192.168.1.110 - - [12/Feb/2018 21:36:52] "GET /task/list?state=all HTTP/1.1" 200 -
192.168.1.110 - - [12/Feb/2018 21:36:52] "GET /task/tid/f95b23efcc1c39d1bfed70f91217ad23a9bf35cc/status HTTP/1.1" 200 -
192.168.1.110 - - [12/Feb/2018 21:36:59] "GET /task/list?state=all HTTP/1.1" 200 -
192.168.1.110 - - [12/Feb/2018 21:37:06] "GET /task/tid/f95b23efcc1c39d1bfed70f91217ad23a9bf35cc/status HTTP/1.1" 200 -
  INFO -    task.py:82  - Task finishes, url - https://www.youtube.com/watch?v=8w2lNpixqOc(517415b65368cd8b43db4fcf35009143a95a1334)
  INFO -  worker.py:130 - Terminating Process ...
 ERROR -    task.py:257 - Task does not active, tid=517415b65368cd8b43db4fcf35009143a95a1334
 ERROR -    task.py:257 - Task does not active, tid=517415b65368cd8b43db4fcf35009143a95a1334
```

**Comment by d0u9** [comment]
@jaredrsowers I tested in virtual machine, and it works fine.

**Comment by carlosedp** [comment]
Yes, latest commit from that time. Any fixes related to this in past 20 days?

**Comment by d0u9** [comment]
No, I made no changes since 28 days ago.

**Comment by frankforpresident** [comment]
Seeing the same issue when running in a docker container

  INFO -  worker.py:130 - Terminating Process ...                                                                                                                                                                       
 DEBUG -      db.py:148 - db finish_task()                                                                                                                                                                              
 ERROR -    task.py:257 - Task does not active, tid=0f3301b58d43f2e54f195d4cdb8425ba1a22be72                                                                                                                            
 ERROR -    task.py:257 - Task does not active, tid=0f3301b58d43f2e54f195d4cdb8425ba1a22be72                                                                                                                            
Traceback (most recent call last):                                                                                                                                                                                      
  File "/usr/local/lib/python3.6/runpy.py", line 193, in _run_module_as_main                                                                                                                                            
    "__main__", mod_spec)                                                                                                                                                                                               
  File "/usr/local/lib/python3.6/runpy.py", line 85, in _run_code                                                                                                                                                       
    exec(code, run_globals)                                                                                                                                                                                             
  File "/usr/src/youtube_dl_webui/youtube_dl_webui/__main__.py", line 29, in <module>                                                                                                                                   
    youtube_dl_webui.main()                                                                                                                                                                                             
  File "/usr/src/youtube_dl_webui/youtube_dl_webui/__init__.py", line 27, in main                                                                                                                                       
    core.start()                                                                                                                                                                                                        
  File "/usr/src/youtube_dl_webui/youtube_dl_webui/core.py", line 289, in start                                                                                                                                         
    self.msg_mgr.run()                                                                                                                                                                                                  
  File "/usr/src/youtube_dl_webui/youtube_dl_webui/msg.py", line 78, in run                                                                                                                                             
    cb(svr, evnt, data, arg)                                                                                                                                                                                            
  File "/usr/src/youtube_dl_webui/youtube_dl_webui/core.py", line 215, in event_progress                                                                                                                                
    cls._task_mgr.progress_update(tid, data)                                                                                                                                                                            
  File "/usr/src/youtube_dl_webui/youtube_dl_webui/task.py", line 266, in progress_update                                                                                                                               
    raise TaskInexistenceError('task does not exist')                                                                                                                                                                   
youtube_dl_webui.utils.TaskInexistenceError: 'Task does not exist'

**Comment by d0u9** [comment]
@frankforpresident Can you reproduce this error?

**Comment by d0u9** [comment]
Fixed by 96062d9.
